README.md

```diff
@@ -74,6 +74,8 @@ docker run -d -p 5000:5000 vampi_docker:latest
 docker-compose up -d
 ~~~~
 
+
```

Comment:
Should include instructions for running integration tests, API endpoint tests, and expected test coverage metrics (e.g., minimum coverage percentage required for PRs to be accepted).